namespaces/live.cloud-platform.service.justice.gov.uk/court-probation-dev/resources/court-cases-queue.tf

```diff
@@ -0,0 +1,114 @@
+module "court-cases-queue" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-sqs?ref=5.0.0"
+
+  # Queue configuration
+  sqs_name                    = "court-cases"
+  encrypt_sqs_kms             = "true"
+  message_retention_seconds   = 1209600
+  visibility_timeout_seconds  = 120
+  fifo_queue                  = "true"
+  content_based_deduplication = "true"
+
+  redrive_policy = jsonencode({
+    deadLetterTargetArn = module.court-cases-dlq.sqs_arn
+    maxReceiveCount     = 3
+  })
+
+  # Tags
+  business_unit          = var.business_unit
+  application            = "court-case-matcher"
+  is_production          = var.is_production
+  team_name              = var.team_name
+  namespace              = var.namespace
+  environment_name       = var.environment-name
+  infrastructure_support = var.infrastructure_support
+
+  providers = {
+    aws = aws.london
+  }
+}
+
+# resource "aws_sqs_queue_policy" "court-cases-queue-policy" {
```

Comment:
Please remove this commented out section for now. We can add it in when required and check it then.